AmericanMedicalAssociation/AMA-style-guide
# EWL 3119 - Multi step form

## Please view #61 before this PR

Ticket: https://issues.ama-assn.org/browse/EWL-3119

To test:
- Go to Organisms > Sections > Multi Step Form http://localhost:3000/?p=organisms-multi-step-form
- Walkthrough the form to ensure that validation happens on the current set of items before moving on to the next step.
- That you can go back to any step prior that has been completed.
- Check the breakpoint styling to verify the form is usable at any size.
